src/SuplaBundle/Entity/Schedule.php

```diff
@@ -115,6 +115,12 @@ class Schedule {
      */
     private $caption;
 
+    /**
+     * @ORM\Column(name="retry", type="boolean", nullable=false)
```

Comment:
Add default=1 here so the schema is updated accordingly. The default assignment below is not sufficient.